package-lock.json

```diff
@@ -8056,8 +8056,7 @@
     "openapi-types": {
       "version": "1.3.5",
       "resolved": "https://registry.npmjs.org/openapi-types/-/openapi-types-1.3.5.tgz",
-      "integrity": "sha512-11oi4zYorsgvg5yBarZplAqbpev5HkuVNPlZaPTknPDzAynq+lnJdXAmruGWP0s+dNYZS7bjM+xrTpJw7184Fg==",
-      "dev": true
```

Comment:
For some reason the "dev" property did not get removed when I moved the dependencies. I tried a few different things and the `openapi-types` module wouldn't even get removed when I ran `npm remove openapi-types`.

I manually edited this file to remove this flag. I primarily use yarn, so am no longer in touch with the nuances of npm.